File: create.py
def main():
    f = open("isEven.js", "w+")

    f.write("""function issEven(number){
    switch(number){
    """)
    
    for i in range(5001):
        f.write(f'        case {i}:\n')
        if (i % 2):
            f.write('            return false;\n')
        else:
            f.write('            return true;\n')
        f.write('            break;\n')
    
    f.write("""           default:
               return "Tired. Figure it out on your own.";
               break;
       }
   }
  module.exports.isEven = isEven;""")

if __name__ =="__main__":
    main()

File: create.cpp
#include <iostream>
#include <fstream>
using namespace std;

int main() {
  // Create and open file
  ofstream MyFile("isOdd.js");

  // Write to the file
  MyFile << "function issEven(number){\n"
            "    switch(number){\n";

  for (int i=0; i <= 5000; i++) {
    MyFile << "        case " + std::to_string(i) + ":\n";
    if (!(i % 2)) {
        MyFile << "            return false;\n";
    } else {
        MyFile << "            return true;\n";
    }
    // MyFile << "            break;\n";
  }

  MyFile << "           default:\n"
            "             return \"Tired. Figure it out on your own.\"\n"
            // "             break;\n"
            "  }\n"
            "}\n"
  "module.exports.isOdd = isOdd;\n";

  // Close the file
  MyFile.close();
}
